docs/conf.py

```diff
@@ -12,18 +12,23 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-import os
-import sys
-sys.path.insert(0, os.path.abspath('../'))
-sys.path.insert(0, os.path.abspath('.'))
-sys.path.insert(0, os.path.abspath('./'))
+import datetime
+
+from ansible_runner.__main__ import VERSION
```

Comment:
If it's expected to be installed, just use `importlib.metadata.version('ansible-runner')` instead of importing it.

Reply:
I was doing that originally, but ran into an issue because we use Python 3.6 by default in CI. I think I could get around it by setting `basepython` for the `tox` environment, but this seemed easier (and it's a bit of dogfooding our API for getting version information).